tests/unit/commands/buildcmd/test_command.py

```diff
@@ -284,3 +285,29 @@ def test_two_functions_with_default_image(self):
             ["Function1=image1", "Function2=image2", "image3"],
             {"Function1": "image1", "Function2": "image2", None: "image3"},
         )
+
+
+@patch("samcli.commands.build.click_container.ContainerOptions")
+class TestContainerOptionsSucceeds(TestCase):
```

Comment:
Nit: Since this is in its own file, lets match that with our unit tests as well. This is will make it consistent with all our other tests in the repo and make it easier to know where to update tests in the future when things change.

Reply:
Sounds good